doc/manpages/dinit-service.5.m4

```diff
@@ -182,13 +182,16 @@ Variable substitution is not performed on the \fBenv\-file\fR property value its
 If the path is not absolute, it is resolved relative to the directory containing the service
 description.
 .TP
-\fBrestart\fR = {yes | true | no | false}
+\fBrestart\fR = {yes | true | on-failure | no | false}
 Indicates whether the service should automatically restart if it stops, including due to
-unexpected process termination or a dependency stopping.
+unexpected process termination or a dependency stopping. "on-failure" is special option
+and indicates auto-restart should happen only when exit code of process is not equal to 0.
```

Comment:
... "when _the_ exit code of _the service_ process is not equal to 0".

Use semicolon not full stop between "dependency stopping" and ""on-failure"".

What if it's not a process-based service?

Reply:
Also: how does this apply to `scripted` services?